force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -300,15 +394,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Map<String, Integer> addItemsToMap() {
         // Initialize an empty Map
-
+        Map<String, Integer> fruitInventory = new Map<String, Integer>();
         // Add key-value pairs to the Map
-        // add Apples => 5
-        // add Oranges => 10
-        // add Bananas => 15
-        // add Pears => 20
-        // add Grapes => 25
+        fruitInventory.put('Apples', 5);// add Apples => 5
+        fruitInventory.put('Oranges', 10);// add Oranges => 10
+        fruitInventory.put('Bananas', 15);// add Bananas => 15
+        fruitInventory.put('Pears', 20);// add Pears => 20
+        fruitInventory.put('Grapes', 25);// add Grapes => 25
 
-        return null; // Replace null with the variable you used to store the result
+        return fruitInventory; // Replace null with the variable you used to store the result
```

Comment:
Good solution. Another approach is to use the map Constructor. 

 return new Map<String, Integer>{
                'Apples' => 5,
                'Oranges' => 10,
                'Bananas' => 15,
                'Pears' => 20,
                'Grapes' => 25
        };